Doc/whatsnew/3.11.rst

```diff
@@ -213,6 +213,11 @@ sqlite3
   :meth:`~sqlite3.Connection.set_authorizer`.
   (Contributed by Erlend E. Aasland in :issue:`44491`.)
 
+* :meth:`~sqlite3.Connection.create_collation` now accepts UTF8 collation
+  names.  Collation names with invalid characters now raise
+  :exc:`UnicodeEncodeError` instead of :exc:`sqlite3.ProgrammingError`.
+  (Contributed by Erlend E. Aasland in :issue:`44688`.)
+
```

Comment:
```suggestion
* Collation name :meth:`~sqlite3.Connection.create_collation` can now 
  contain any Unicode character.  Collation names with invalid characters
  now raise :exc:`UnicodeEncodeError` instead of :exc:`sqlite3.ProgrammingError`.
  (Contributed by Erlend E. Aasland in :issue:`44688`.)

```